samples/bluetooth/central_otc/src/main.c

```diff
@@ -0,0 +1,585 @@
+		discover_params.type = BT_GATT_DISCOVER_CHARACTERISTIC;
+
+		err = bt_gatt_discover(conn, &discover_params);
+		if (err) {
+			printk("Discover failed (err %d)\n", err);
+		}
+	} else if (!bt_uuid_cmp(discover_params.uuid, BT_UUID_OTS_ACTION_CP)) {
+		atomic_set_bit(&discovery_state, DISC_OTS_ACTION_CP);
+		otc.oacp_handle = bt_gatt_attr_value_handle(attr);
+		memcpy(&uuid, BT_UUID_OTS_LIST_CP, sizeof(uuid));
+		discover_params.uuid = &uuid.uuid;
+		discover_params.start_handle = attr->handle + 1;
+		discover_params.type = BT_GATT_DISCOVER_CHARACTERISTIC;
+
+		err = bt_gatt_discover(conn, &discover_params);
+		if (err) {
+			printk("Discover failed (err %d)\n", err);
+		}
+	} else if (!bt_uuid_cmp(discover_params.uuid, BT_UUID_OTS_LIST_CP)) {
+		atomic_set_bit(&discovery_state, DISC_OTS_LIST_CP);
+		otc.olcp_handle = bt_gatt_attr_value_handle(attr);
+	} else {
+		return BT_GATT_ITER_STOP;
+	}
+
+	if (is_discovery_complete()) {
+		printk("Discovery complete for OTS Client\n");
+		subscribe_func();
+		bt_ots_client_register(&otc);
+	}
+
+	return BT_GATT_ITER_STOP;
+}
+
+static void connected(struct bt_conn *conn, uint8_t err)
+{
+	char addr[BT_ADDR_LE_STR_LEN];
+
+	bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
+
+	if (err) {
+		printk("Failed to connect to %s (%u)\n", addr, err);
+
+		bt_conn_unref(default_conn);
+		default_conn = NULL;
+
+		start_scan();
+		return;
+	}
+
+	if (conn != default_conn) {
+		return;
+	}
+
+	printk("Connected: %s\n", addr);
+
+	if (conn == default_conn) {
+		memcpy(&uuid, BT_UUID_OTS, sizeof(uuid));
+		discover_params.uuid = &uuid.uuid;
+		discover_params.func = discover_func;
+		discover_params.start_handle = BT_ATT_FIRST_ATTRIBUTE_HANDLE;
+		discover_params.end_handle = BT_ATT_LAST_ATTRIBUTE_HANDLE;
+		discover_params.type = BT_GATT_DISCOVER_PRIMARY;
+
+		err = bt_gatt_discover(default_conn, &discover_params);
+		if (err) {
+			printk("Discover failed(err %d)\n", err);
+			return;
+		}
+	}
+}
+
+static void disconnected(struct bt_conn *conn, uint8_t reason)
+{
+	char addr[BT_ADDR_LE_STR_LEN];
+
+	if (conn != default_conn) {
+		return;
+	}
+
+	bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
+
+	printk("Disconnected: %s (reason 0x%02x)\n", addr, reason);
+
+	bt_conn_unref(default_conn);
+	default_conn = NULL;
+	discovery_state = ATOMIC_INIT(0);
+	start_scan();
+}
+
+BT_CONN_CB_DEFINE(conn_callbacks) = {
+	.connected = connected,
+	.disconnected = disconnected,
+};
+
+static void on_obj_selected(struct bt_ots_client *ots_inst, struct bt_conn *conn, int err)
+{
+	printk("Current object selected cb %d\n", err);
+
+	if (err == BT_GATT_OTS_OLCP_RES_OPERATION_FAILED) {
+		printk("BT_GATT_OTS_OLCP_RES_OPERATION_FAILED %d\n", err);
+	} else if (err == BT_GATT_OTS_OLCP_RES_OUT_OF_BONDS) {
+		printk("BT_GATT_OTS_OLCP_RES_OUT_OF_BONDS %d. Select first instead\n", err);
+		bt_ots_client_select_first(&otc, default_conn);
+	}
+	memset(obj_data_buf, 0, OBJ_MAX_SIZE);
```

Comment:
```suggestion
	}
	
	memset(obj_data_buf, 0, OBJ_MAX_SIZE);
```

We also generally leave an empty line after a closing curly bracket

Reply:
Add empty line to all closing curly bracket.